erpnext_documentation/www/docs/v13/user/manual/en/human-resources/employee-grievance.md

```diff
@@ -0,0 +1,41 @@
+<!-- add-breadcrumbs -->
+# Employee Grievance
+
+> Introduced in version-13
+
+**An employee grievance is a concern, problem, or complaint raised by an employee with what he expects from the company(or their individuals) and its management. Something has made them feel dissatisfied, and they believe it is unfair and/or unjust to them. It may or may not be justified but needs to be tackled very carefully.**
```

Comment:
An Employee Grievance is a concern, problem, or complaint raised by an employee with what he expects from the company(or their individuals) and its management. Something has made them feel dissatisfied, and they believe it is unfair and/or unjust to them. It may or may not be justified but needs to be tackled very carefully.

Reply:
done